File: ejemplos/e3-csharp/Models/Classroom.cs
using System.ComponentModel.DataAnnotations;

namespace e3_csharp.Models
{
    public class Classroom
    {
        [Key]
        public int Id { get; set; }

        [Required]
        [StringLength(50)]
        public string Name { get; set; } = string.Empty;

        [Required]
        public int Capacity { get; set; }

        [StringLength(200)]
        public string Description { get; set; } = string.Empty;

        [Required]
        public bool IsActive { get; set; } = true;
    }
}

File: ejemplos/e6-client-react/src/types/classroom.ts
export interface Classroom {
    id: number;
    name: string;
    capacity: number;
    location: string;
    isActive: boolean;
}

export interface CreateClassroomDto {
    name: string;
    capacity: number;
    location: string;
    isActive: boolean;
}

export interface UpdateClassroomDto {
    id: number;
    name: string;
    capacity: number;
    location: string;
    isActive: boolean;
} 